Angela/pcx 3093 update logs examples for consistency (#3131)
cloudflare/cloudflare-docs: products/logs/src/content/logpull/requesting-logs/index.md

`logs/received`

```bash

> https://api.cloudflare.com/client/v4/zones/<ZONE_ID>/logs/received?start=<unix|rfc3339>&end=<unix|rfc3339>[&count=<int>][&sample=<float>][&fields=<fields>][&timestamps=<string>][&CVE-2021-44228=<boolean>]

Reviewer: nit:
```suggestion
https://api.cloudflare.com/client/v4/zones/<ZONE_ID>/logs/received?start=<unix|rfc3339>&end=<unix|rfc3339>[&count=<int>][&sample=<float>][&fields=<FIELDS>][&timestamps=<string>][&CVE-2021-44228=<boolean>]
```

```bash
curl -s \
    -H "X-Auth-Email: <EMAIL>" \
    -H "X-Auth-Key: <API_KEY>" \

>     "https://api.cloudflare.com/client/v4/zones/<ZONE_ID>/logs/received?start=2017-07-18T22:00:00Z&end=2017-07-18T22:01:00Z&count=1&fields=RayID,ClientIP"

Reviewer: nit, maybe use default fields:
```suggestion
    "https://api.cloudflare.com/client/v4/zones/<ZONE_ID>/logs/received?start=2017-07-18T22:00:00Z&end=2017-07-18T22:01:00Z&count=1&fields=ClientIP,ClientRequestHost,ClientRequestMethod,ClientRequestURI,EdgeEndTimestamp,EdgeResponseBytes,EdgeResponseStatus,EdgeStartTimestamp,RayID"
```

```

> `logs/rayids`

Reviewer 2: ```suggestion
`logs/rayids/<RAY_ID>`
```

Unless specified in the <em>fields parameter</em>, the API returns a limited set of log fields. This default field set may change at any time. The list of all available fields is at:

`https://api.cloudflare.com/client/v4/zones/<ZONE_ID>/logs/received/fields`

The order in which fields are specified doesn't matter, and the order of fields in the response is not specified.

Using <em>Bash</em> subshell and <em>jq</em>, you can download the logs with all available fields without manually copying and pasting the fields into the request. For example:

```bash
curl -s \
    -H "X-Auth-Email: <EMAIL>" \
    -H "X-Auth-Key: <API_KEY>" \
    "https://api.cloudflare.com/client/v4/zones/<ZONE_ID>/logs/received?start=2017-07-18T22:00:00Z&end=2017-07-18T22:01:00Z&count=1&fields=$(curl -s -H "X-Auth-Email: <EMAIL>" -H "X-Auth-Key: <API_KEY>" "https://api.cloudflare.com/client/v4/zones/<ZONE_ID>/logs/received/fields" | jq '. | to_entries[] | .key' -r | paste -sd "," -)"
```

> *See [HTTP request fields](/reference/log-fields/#http-requests)* for the currently available fields.

Reviewer 2: ```suggestion
*Refer to [HTTP request fields](/reference/log-fields/#http-requests)* for the currently available fields.
```
